src/js/player.js

```diff
@@ -2481,23 +2481,14 @@ class Player extends Component {
 }
 
 /*
- * Global player list
- *
- * @type {Object}
- */
-Player.players = {};
-
-/*
  * Player instance options, surfaced using options
  * options = Player.prototype.options_
  * Make changes in options, not here.
- * All options should use string keys so they avoid
- * renaming by closure compiler
  *
  * @type {Object}
  * @private
  */
-Player.prototype.options_ = globalOptions;
+Player.prototype.options_ = mergeOptions(defaultOptions);
```

Comment:
This is really what the default options are. The player prototype options. I think we could just move the default-options file contents here and then expose them on videojs later when player.js is imported into video.js.